main.py

```diff
@@ -58,45 +65,44 @@ def _run(self):
                 continue
 
             # Yeey this post probably isn't breaking the rules 🌈
-            comment = submission.reply(self.last_fm.get_artist_reply(artist_name))
-            comment.mod.distinguish("yes", sticky=True)
-            self.database.save_submission(submission)
+            self._reply_with_sticky_post(submission, self.last_fm.get_artist_reply(artist_name))
 
     def _archive_submission_exceeding_threshold(self, submission, service, threshold, count):
-        reply = '''
-All apologies /u/{} but your post has been automatically removed because the artist has too many {}. The maximum is {:,}, this link has {:,}.
-If you think this is in error, please [contact the mods](https://www.reddit.com/message/compose?to=/r/listentothis&subject=Post+removed+in+error.&message=https://redd.it/{}). 
+        reply = templates.submission_exceeding_threshold.format(
+            submission.author.name, service, threshold, count, submission.id
+        )
 
-If you're new to the subreddit, please read the full list of removal reasons.
-
-_Don't blame me,_ [_I'm just a bot_](https://www.youtube.com/watch?v=jqaweMZv4Og)|[_Bugs & Code_](https://github.com/martijnboers/BlotterTrax)
-        '''.format(submission.author.name, service, threshold, count, submission.id)
-
-        comment = submission.reply(reply)
-        comment.mod.distinguish("yes", sticky=True)
-
-        submission.mod.remove()
-        self.database.save_submission(submission)
+        self._reply_with_sticky_post(submission, reply, True)
 
     def _archive_text_post_without_discussion_tag(self, submission):
-        reply = '''
-All apologies /u/{} but your post has been automatically removed because it is a text post without the [Discussion] tag 
-
-If you're new to the subreddit, please read the full list of removal reasons.
+        reply = templates.submission_missing_discussion_tag.format(submission.author.name, submission.id)
 
-_Don't blame me,_ [_I'm just a bot_](https://www.youtube.com/watch?v=jqaweMZv4Og)|[_Bugs & Code_](https://github.com/martijnboers/BlotterTrax)
-                '''.format(submission.author.name, submission.id)
+        self._reply_with_sticky_post(submission, reply, True)
 
-        comment = submission.reply(reply)
+    def _reply_with_sticky_post(self, submission, reply_text, remove=False):
+        comment = submission.reply(reply_text)
         comment.mod.distinguish("yes", sticky=True)
 
-        submission.mod.remove()
+        if remove is True:
+            submission.mod.remove()
 
         self.database.save_submission(submission)
 
     @staticmethod
-    def _extract_artist_post_title(post_title):
-        return post_title.split(' -')[0]
+    def _get_artist_name_from_submission_title(post_title):
+        artist = re.search(r'(?P<artist>.+?) \s*(-|—)\s*', post_title, re.IGNORECASE)
+        feature_artist = re.search(
+            r'(?P<artist>.+?)\s*(&|ft|ft\.|feature|featuring)\s*(?P<featureArtist>.+?)\s*(-|—)\s*',
```

Comment:
Think this can simplify to..

Edit: On second thought, might want to expand upon it a bit to allow us to detect "feat" and  "feat."

```suggestion
            r'(?P<artist>.+?)\s*(&|feat.?|ft\.?|feature|featuring)\s*(?P<featureArtist>.+?)\s*(-|—)\s*',
```